calico-enterprise_versioned_docs/version-3.17/operations/cnx/configure-identity-provider.mdx

```diff
@@ -42,214 +43,242 @@ When configuring your cluster, you may be asked for the following inputs:
+
+    ```yaml noValidation
+    apiVersion: operator.tigera.io/v1
+    kind: Authentication
+    metadata:
+    name: tigera-secure
+    spec:
+    # This indicates where the Manager UI can be accessed from the browser. Include the port only if the the manager UI is not running on port 443.
+    managerDomain: https://<manager-host>:<port>
+    ldap:
+    # The host and port of the LDAP server. Example: ad.example.com:636.
+    host: <ldap-host>:<ldap-port>
+    # (optional) StartTLS whether to enable the startTLS feature for establishing TLS on an existing LDAP session.
+    # If true, the ldap protocol is used and then issues a StartTLS command, otherwise, connections will use
+    # the ldaps: protocol.
+    startTLS: true
+
+    # User entry search configuration.
+    userSearch:
+    # To start the user search from. Example: "cn=users,dc=example,dc=com".
+    baseDN: <base-dn>
+
+    # Optional filter to apply when searching the directory. Example: "(objectClass=posixAccount)"
+    filter: <filter>
+
+    # A mapping of attributes to the username. This value can be used for applying RBAC to a user.
+    # Default: uid
+    nameAttribute: <name-attribute>
+
+    # (Optional) Group search configuration. This value can be used to apply RBAC to a user group.
+    groupSearch:
+
+    # BaseDN to start the search from. Example: "cn=groups,dc=example,dc=com".
+    baseDN: <base-dn>
+
+    # Optional filter to apply when searching the directory. Example: "(objectClass=posixGroup)"
+    filter: <filter>
+
+    # A mapping of attributes to the group name. This value can be used for applying RBAC to a user group. Example: "cn".
+    nameAttribute: <name-attribute>
+
+    # Following list contains field pairs that are used to match a user to a group. It adds an additional
+    # requirement to the filter that an attribute in the group must match the user's attribute value.
+    userMatchers:
+    - userAttribute: <user-attribute>
+    groupAttribute: <group-attribute>
+    ```
+
+    1. Apply the secret to your cluster with your LDAP credentials. To obtain the values, consult the documentation of your provider.
+
+    ```yaml
+    apiVersion: v1
+    kind: Secret
+    metadata:
+    name: tigera-ldap-credentials
+    namespace: tigera-operator
+    data:
+    bindDN: <your-base64-bind-dn>
+    bindPW: <your-base64-bind-password>
+    rootCA: <your-base64-ca-cert-pem>
+    ```
+
+    ### Troubleshooting LDAP
+
+    **Problem**: Error on Manager UI: 502 bad gateway nginx
+
+    **Solution/workaround**: If you have many groups of users, you may need to add filters in the AD configuration to reduce the number of groups that are passed to {{prodname}}. The issue is that nginx ingress cannot handle the length of the token. To solve this issue, apply a filter: `(&(objectClass=group)(|(cn:dn:=HwInf-K8S-Support)(cn:dn:=HWInf-CISD)))` as shown in the following example.
+
+    ```yaml
```

Comment:
Like all other yamls, this is malformatted.

I think that the mentions of "HwInf-K8S-Support" and "HWInf-CISD"  should be censored. For example "example-dn1" and "example-dn2"

Reply:
Fixed.